### DISCUSSION TITLE
Document other backup solutions

### DISCUSSION THREAD

**Discussion by jonathan-s** [Archmonger/django-dbbackup#582] [Feature Requests]
- Barman
- tarsnap

And how these can work together.

**Comment by bittner** [comment]
* [runrestic](https://pypi.org/project/runrestic/) – _one of the more promising [attempts to provide restic](https://pypi.org/search/?q=restic) for Python_

**Comment by Archmonger** [comment]
Closing since this adds additional maintenance burden (investigation / deprecation checks / etc).

Alternatives would better be tracked by a third-party, (e.g. [awesome-database](https://github.com/agarcialeon/awesome-database))
